lms/static/sass/views/_program-details.scss

```diff
@@ -225,24 +258,6 @@
     }
 
     .course-list-headings {
-        width: 700px;
-
-        .divider {
-            margin-left: 0;
-            margin-bottom: 20px;
-            background-color: $divider-color;
-            margin-top: 5px;
-            height: 3px;
-            width: 315px;
-            @media(min-width: $bp-screen-sm) {
-                width: 550px;
-            }
-            @media(min-width: $bp-screen-md) {
-                width: 700px;
-            }
-            border: none;
-        }
-
         .motivating-section {
             font-size: 0.9375em;
             margin-left: 15px;
```

Comment:
This one isn't yours, but this should also use bi-app. Maybe you could change all the occurrences in this file.